fsharp-backend/src/LibExecution/Interpreter.fs

```diff
@@ -72,10 +76,11 @@ let rec eval' (state : ExecutionState) (st : Symtable) (e : Expr) : DvalTask =
       // We ignore incompletes but not error rail.
       // CLEANUP: Other places where lists are created propagate incompletes
       // instead of ignoring, this is probably a mistake.
-      let! results = Ply.List.mapSequentially (eval state st) exprs
+      let! results = exprs |> Ply.List.mapSequentially (eval state st)
```

Comment:
Hope this isn't too nitpicky - I find it much easier to read relying on pipes.

Reply:
I would prefer we revert these changes. I'm not against pipeing but I find that it's more confusing when there's only one stage in the pipe.